kolibri/core/assets/src/vue/textbox.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <ui-textbox
+    @focus="$emit('focus')"
+    @input="updateText()"
+    v-model = "currentText"
```

Comment:
^ Needed to be able to accept v-model input

https://vuejs.org/v2/guide/components.html#Form-Input-Components-using-Custom-Events